airbyte-integrations/connector-templates/source-java-jdbc/README.md

```diff
@@ -0,0 +1,68 @@
+# Source {{titleCase name}}
+
+This is the repository for the {{titleCase name}} source connector in Java.
```

Comment:
JDBC is for database only. So this is for database only Java source, right?

Reply:
yes